### PR TITLE
Manually screaming is now audible

### PR DESCRIPTION
**Pull request by yacabo**
# General Documentation

### Intent of your Pull Request

Allows players to create an audible scream when using the *scream emote instead of it only being triggered by code. It has a cooldown of ten seconds so cannot be spammed that much, do feel free to complain if you want it higher

### Why is this change good for the game?

More noises makes the game cooler. 

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
This will be the basis of the Wiki entry for your PR, and more information / detail is better for Wiki editors to integrate.

Players can audibly scream now

### What should players be aware of when it comes to the changes your PR is implementing?

Players can audibly scream now

### What general grouping does this PR fall under? 
For example, Engineering changes, Medical rebalancing, TEG tweaks, etc.?

Emotes

### Are there any aspects of the PR that you would like us not to mention on the Wiki?

Nyet

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
For example, "This new chem will heal X brute damage". 

Nah

# Changelog

:cl:  
soundadd: You can scream now
/:cl:

### CODE REVIEW

**Comment by yacabo** [comment]
idk why this wasn't done sooner and I'm worried I should

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -66,7 +66,7 @@
 	key_third_person = "screams"
 	message = "screams!"
 	emote_type = EMOTE_AUDIBLE
-	only_forced_audio = TRUE
+	cooldown = 10 SECONDS
```
Comment (alexkar598) [inline code/modules/mob/living/carbon/human/emote.dm:69]: Just pointing this out for the fucktards that will downvote the pr because "you can spam it"

**Comment by maxion12345** [comment]
Might downvote because I can't spam it

**Comment by Tipy1802** [comment]
For those who don't think the 10s cooldown is not enough, I would like to remind you that there is also a cooldown where if you spam and emote too much you get muted by the server
